### PR TITLE
Leaderboard fixes and announcement

### LINKED ISSUE #36: Add date announcement

### LINKED ISSUE #44: Fully seperate SRC and DIST

### LINKED ISSUE #45: Use webpack devtools to make source map for debugging

### LINKED ISSUE #47: Figure out w3m modal error on devserver
This: https://github.com/WalletConnect/walletconnect-docs/issues/762

(This doesn't affect production)

### PR DESCRIPTION
**Pull request by jMyles**
Makes leaderboard work; integrates WAGMI for reads and some writes; cleans up webpack integration.

fixes #47 
fixes #44 
fixes #45
fixes #36

### CODE REVIEW

**Comment by kubic71** [comment]
image

Still getting the same error with the `setMinPreset` and other javascript functions not being loaded correctly.

Clicking on the buttons also doesn't do anything:
image

File: src/js/index.js [hunk 1]
```diff
@@ -171,44 +221,37 @@ document.addEventListener("DOMContentLoaded", () => {
...
-        })
-        .catch((error) => console.error);
-}
+    let contributionsByAddress = getContributionsByAddress(contributionsMetadata)
+    let leaders = getTopContributions(contributionsByAddress)
+
+    // Get the table body
+    const leaderboardTable = document.getElementById('leaderboard-table');
+    const leaderboardTableBody = leaderboardTable.childNodes[3]
+    const leaderRows = leaderboardTableBody.getElementsByTagName("tr");
```
Comment (kubic71) [inline src/js/index.js:237]: Getting different results for `leaderboard-table` when using development build vs. prod build.

When running `npm run devserver`, I get `<table />` with 5 child nodes:
image

The production build has the table with ordinary `2` child nodes:
image

So getting to the body with `index=3` won't work.
It is also broken on the currently deployed version.

Comment (jMyles) [reply src/js/index.js:237]: Whew... Not sure why that is.

We probably just need to operate on the ID of a lower-nested node.

If it works for you on devserver., then I think let's still merge this and work from there.

Comment (kubic71) [reply src/js/index.js]: Ok, creating new issue for that.

**Pull request merged**